boot/debug.go

```diff
@@ -28,24 +28,43 @@ import (
 
 // DumpBootVars writes a dump of the snapd bootvars to the given writer
 func DumpBootVars(w io.Writer, dir string, uc20 bool) error {
-	bloader, err := bootloader.Find(dir, nil)
-	if err != nil {
-		return err
+	opts := &bootloader.Options{
+		NoSlashBoot: dir != "" && dir != "/",
+	}
+	switch dir {
+	// is it any of the well-known UC20 boot partition mount locations?
+	case InitramfsUbuntuBootDir:
```

Comment:
I think we should do this also if  NoSlashBoot is false, and /var/lib/snapd/modeenv exitsts

Reply:
Good idea, I'll push a patch.